types/jws/index.d.ts

```diff
@@ -28,7 +28,7 @@ export function verify(signature: string, algorithm: Algorithm, secretOrKey: str
  * (Synchronous) Returns the decoded header, decoded payload,
  * and signature parts of the JWS Signature.
  */
-export function decode(signature: string): Signature;
+export function decode(signature: string): Signature | null;
```

Comment:
What are the cases where this can return `null`?  Can you include a link to the code?